packages/core/src/analytics/posthogAiBetaServerClient/featureFlagEvaluation.ts

```diff
@@ -0,0 +1,57 @@
+import { aiLogger } from "@oakai/logger";
+import { kv } from "@vercel/kv";
+import type { PostHog } from "posthog-node";
+
+const KV_KEY = "posthog-feature-flag-local-evaluation";
+const ONE_DAY = 24 * 60 * 60 * 1000;
+const ONE_MINUTE = 60 * 1000;
+
+const log = aiLogger("analytics:feature-flags");
+
+const setKv = async (response: Response) => {
+  const value = await response.text();
+  await kv.set(KV_KEY, value, { ex: ONE_MINUTE });
+};
+
+const getKv = async () => {
+  const value = await kv.get(KV_KEY);
```

Comment:
And here I think if we use `kv.get<some type>` we can validate it matches the type? I'm not sure where I saw this, but perhaps look up whether this is possible?

Reply:
I did have a look, but I haven't seen it in the docs. I guess for runtime validation it would need to be an argument, and then it could use something like zod. But that could be extra effort for what we actually need